### PR TITLE
Remove files used for running tests

### PR DESCRIPTION
**Pull request by bobleesj**
Same as the title:

Addressing https://github.com/diffpy/diffpy.nmf_mapping/pull/10#pullrequestreview-2231960398

### CODE REVIEW

File: src/diffpy/nmf_mapping/tests/run.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #
 ##############################################################################
 """Convenience module for executing all unit tests with
-python -m \.tests.run
+python -m diffpy.nmf_mapping.tests.run
```
Comment (bobleesj) [inline src/diffpy/nmf_mapping/tests/run.py:16]: flake8 passes.

Comment (sbillinge) [reply src/diffpy/nmf_mapping/tests/run.py:16]: This is the wrong syntax here as we are moving over to using pytest to run tests.  Please look in cookicutter or one of the other projects

**Review by sbillinge — COMMENTED** [review]
Pls e comments

File: src/diffpy/nmf_mapping/tests/run.py (hunk 1 above)
Comment (bobleesj) [reply src/diffpy/nmf_mapping/tests/run.py:16]: From the cookiecutter repo, I see it is the latest `tests/run.py` file:

https://github.com/Billingegroup/cookiecutter/blob/main/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/%7B%7B%20cookiecutter.package_dir_name%20%7D%7D/tests/run.py

I checked diffpy.structure

https://github.com/diffpy/diffpy.structure/blob/main/src/diffpy/structure/tests/run.py

but it seems even older.

What is the "latest" one?

**Comment by bobleesj** [comment]
@sbillinge  Please see https://github.com/diffpy/diffpy.snmf/pull/72

File: src/diffpy/nmf_mapping/tests/run.py (hunk 1 above)
Comment (sbillinge) [reply src/diffpy/nmf_mapping/tests/run.py:16]: I think we just delete this run.py?   Then in the ci (and when we run it locally) we just run `pytest` or `coverage`

**Review by sbillinge — COMMENTED** [review]
pls see inline

**Review by bobleesj — COMMENTED** [review]
Files used for running tests removed.

This will be the last edit in a while. 

Now, other diffpy projects will be priotized.

**Comment by sbillinge** [comment]
@bobleesj this is failing tests.  Do you want me to merge it and get tests passing on a separate PR?

**Comment by bobleesj** [comment]
> @bobleesj this is failing tests. Do you want me to merge it and get tests passing on a separate PR?

Yes, please merge this one. 

For `diffpy.nmf_mapping`, 3 out of 4 tests have not been passing for some reason from the start. Probably we shall look into this later. Made an issue for now. 

https://github.com/diffpy/diffpy.nmf_mapping/issues/14

**Pull request merged**